cmd/openapi-gen/examples/pkg/api/v1/definition.go

```diff
@@ -0,0 +1,66 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/caicloud/nirvana/definition"
+)
+
+// Desc is global descriptor of api
+var Desc = definition.Descriptor{
+	Path:        "/api/v1",
+	Definitions: []definition.Definition{},
+	Consumes:    []string{"application/json"},
+	Produces:    []string{"application/json"},
+	Children: []definition.Descriptor{
+		{
+			Path: "/{target1}/{target2}",
+			Definitions: []definition.Definition{
+				{
+					Method:   definition.Create,
+					Function: Handle,
+					Parameters: []definition.Parameter{
+						{
+							Source: definition.Header,
+							Name:   "User-Agent",
+						},
+						{
+							Source: definition.Query,
+							Name:   "target1",
+						},
+						{
+							Source:  definition.Query,
+							Name:    "target2",
+							Default: false,
+						},
+						{
+							Source: definition.Body,
+							Name:   "app",
+						},
+					},
+					Results: []definition.Result{
+						{Type: definition.Data},
+						{Type: definition.Error},
+					},
+				},
+			},
+		},
+	},
+}
+
+// Application defines application api model
+type Application struct {
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+	Target    string `json:"target"`
+	Target1   int    `json:"target2"`
+	Target2   bool   `json:"target1"`
```

Comment:
wrong

Reply:
Example is copied from @kdada 's example.
It is used to show name in api is generated from json tag.